### PR TITLE
[FIX] deploy: set Content-Security-Policy on static

### PR DESCRIPTION
**Pull request by Julien00859**
The Content-Security-Policy[^1] http header was only set on the response generated by controllers but it was missing from the `/<module>/static/` route.

It is not strictly necessary to set that header on the responses comming from that routes as it is not possible to add new static files or edit existing ones via the interface (not even as admin). Only the developers and system administrator can access those files.

It is also worth mentionning that using the Odoo internal web server to deliver static files is suboptimal. Outside of a dev environment, those files will typically be delivered via a web server[^2] and sysadmins should configure their web server to set the CSP header on static images.

[^1]: https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP
[^2]: https://www.odoo.com/documentation/master/administration/install/deploy.html#serving-static-files-and-attachments

### CODE REVIEW

File: content/administration/install/deploy.rst [hunk 1]
```diff
@@ -439,51 +439,70 @@ Odoo static files are located in each module's :file:`static/` folder, so static
...
-   serve static files via NGINX.
+   .. tabs::
 
-   .. code-block:: nginx
+      .. group-tab:: Debian package
 
-       location @odoo {
-           # copy-paste the content of the / location block
-       }
+         Say Odoo has been installed via the **debian packages** for Community and Enterprise and
```
Comment (joda-odoo) [inline content/administration/install/deploy.rst:482]: > Say Odoo has been installed

`Say Odoo was installed`

Same thing with the part about git

File: content/administration/install/deploy.rst [hunk 2]
```diff
@@ -439,51 +439,70 @@ Odoo static files are located in each module's :file:`static/` folder, so static
...
-   above snippets only highlight two possible configurations and may not be used as-is.
+         .. code-block:: nginx
+
+            root /usr/lib/python3/dist-packages/odoo/addons;
+            try_files $uri @odoo;
+
+      .. group-tab:: Git sources
+
+         Say Odoo has been installed via the **sources**. The two git repositories for Community and
+         Enterprise have been cloned in :file:`/opt/odoo/community` and :file:`/opt/odoo/enterprise`
```
Comment (joda-odoo) [inline content/administration/install/deploy.rst:496]: Maybe both parts should be in the same sentence.

```
Say Odoo was installed via the **source**, and both the Community and Enterprise repositories were cloned in ...
```

File: content/administration/install/deploy.rst [hunk 3]
```diff
@@ -439,51 +439,70 @@ Odoo static files are located in each module's :file:`static/` folder, so static
...
 
-.. example::
-   Say Odoo has been installed via the **debian packages** for Community and Enterprise and the
-   :option:`--addons-path <odoo-bin --addons-path>` is ``'/usr/lib/python3/dist-packages/odoo/addons'``.
+It is recommended to set the ``Content-Security-Policy: default-src 'none'`` header on all images
+delivered by the web server. It is not stricly necessary as users cannot modify/inject content
+inside of modules' :file:`static/` folder and that existing images are final: they don't fetch new
+resources by themselves. Still, it is good hygiene.
+
+Using the above NGINX (https) configuration, the following ``map`` and ``location`` blocks should be
```
Comment (joda-odoo) [inline content/administration/install/deploy.rst:447]: > Using the above NGINX (https) configuration,

In this case the word bellow (ci-dessous) would be more appropriate

```
Using the NGINX (https) configuration provided bellow
```

File: content/administration/install/deploy.rst (hunk 2 above)
Comment (joda-odoo) [inline content/administration/install/deploy.rst:496]:
```suggestion
         Say Odoo has been installed via the **sources**, and both the Community and Enterprise repositories were cloned in :file:`/opt/odoo/community` and :file:`/opt/odoo/enterprise`
```

File: content/administration/install/deploy.rst (hunk 3 above)
[the remark this answers is not in the document]
Comment (Julien00859) [reply content/administration/install/deploy.rst:447]: I'm actually referencing the NGINX configuration shown inside the Https section, which is found earlier (above) in the current document :p The snippet found below is specific to serving static files but work in joint with the https configuration.

File: content/administration/install/deploy.rst (hunk 2 above)
Comment (Julien00859) [reply content/administration/install/deploy.rst]: I reformulated the bit even further using this oxford coma in several other places for consistency

**Review by toaa-odoo — APPROVED** [review]
Cleared up a few typos and phrasing in stead of @odoo/be-doc-review

**Review by Feyensv — APPROVED** [review]
@robodoo delegate+

**Pull request closed without merging**